pkg/gofr/gofr.go

```diff
@@ -224,6 +239,41 @@ func (a *App) Run() {
 	wg.Wait()
 }
 
+func (a *App) hasTelemetry() bool {
+	return a.Config.GetOrDefault("GOFR_TELEMETRY", "true") == defaultTelemetry
+}
```

Comment:
If it's the default value,it should be this no?


```suggestion
func (a *App) hasTelemetry() bool {
	return a.Config.GetOrDefault("GOFR_TELEMETRY", defaultTelemetry) == "true"
}
```